File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
    var answer = 0;
    let parts = binomial.split(" ");
    let a = Number(parts[0]);
    let b = Number(parts[2]);
        switch(parts[1]){
            case "+": answer=a+b;
                      break;
            case "-": answer=a-b;
                      break;
            case "*": answer=a*b;
                      break;
        }
    return answer;
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    answer = 0
    arr = binomial.split(" ")
    if arr[1] == '+' : answer = int(arr[0]) + int(arr[2])
    if arr[1] == '-' : answer = int(arr[0]) - int(arr[2])
    if arr[1] == '*' : answer = int(arr[0]) * int(arr[2])
    return answer